en/04_Changelogs/5.0.0.md

```diff
@@ -1711,9 +1724,22 @@ If you want to get a resource for a _specific_ theme or from somewhere that is n
 - Changed return type for [`Symbiote\QueuedJobs\Services\QueuedJobHandler::write()`](api:Symbiote\QueuedJobs\Services\QueuedJobHandler::write()) from dynamic to `void`
 - Changed parameter type in [`Symbiote\QueuedJobs\Services\QueuedJobHandler::write()`](api:Symbiote\QueuedJobs\Services\QueuedJobHandler::write()) for `$record` from `array` to `Monolog\LogRecord`
 
-#### tractorcow/silverstripe-fluent
+### tractorcow/silverstripe-fluent
 
 - Removed deprecated method `TractorCow\Fluent\Extension\FluentExtension::getLinkingMode()` - use `LocaleInformation()` instead
 - Removed deprecated method `TractorCow\Fluent\Extension\FluentExtension::LocaleLink()` - use `LocaleInformation` instead
 
+</details>
+
+## Full commits list
+
+This release includes a number of bug fixes to improve a broad range of areas. Check the change logs for full details of these fixes split by module. Thank you to the community members that helped contribute these fixes as part of the release!
+
+<details>
+<summary>Reveal full list of commits</summary>
+
+<!--- Changes below this line will be automatically regenerated -->
+
 <!--- Changes above this line will be automatically regenerated -->
+
+</details>
```

Comment:
Are we likely to do this going forward for future changelogs?
Your answer doesn't have any bearing on this PR, I'm just curious.

Reply:
Note also that there's a `changelog` heading that gets put in there automatically that we'll want to remove when it does get generated, since you've added an equivalent heading above this.

Reply:
I don't think this is normally this problematic for minor release changelogs because they tend to be smaller and distinction between the "Release Notes" part and the commit logs is clearer.

Still, if it works well, then maybe we want to adopt this approach going forward.